rotkehlchen/tests/unit/decoders/test_uniswapv2.py

```diff
@@ -1048,3 +1050,48 @@ def test_remove_liquidity_with_weth(database, ethereum_inquirer, ethereum_accoun
         ),
     ]
     assert events == expected_events
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_accounts', [['0x3163Bb273E8D9960Ce003fD542bF26b4C529f515']])
+def test_claim(database, ethereum_inquirer, ethereum_accounts):
```

Comment:
In the last review I said "claim_airdrop"

```suggestion
def test_claim_airdrop(database, ethereum_inquirer, ethereum_accounts):
```